.github/workflows/missing-ops.yml

```diff
@@ -42,31 +43,36 @@ jobs:
         run: poetry -C tests update
       - name: Run the missing op types test
         id: check_missing_optypes
-        run: poetry -C tests run -- cargo test --test integration -- --ignored missing_optypes
-      - name: Set output flags
-        id: check_status
-        if: always()
-        uses: actions/github-script@v7
-        with:
-          script: |
-            const outcome = ${{ steps.check_missing_optypes.outcome != 'success' }};
-            console.log(`The outcome is: ${outcome}`);
-            return outcome
-          result-encoding: string
+        run: |
+          set +e
+          poetry -C tests run -- cargo test --test integration -- --ignored missing_optypes --nocapture --test-threads=1 > missing_optypes.txt
+          if [ $? -eq 0 ]; then
+            echo "The test passed."
+            echo "fail=false" >> $GITHUB_OUTPUT
+          else
+            echo "The test failed with error code $?."
+            echo
+            cat missing_optypes.txt
+            echo "fail=true" >> $GITHUB_OUTPUT
+          fi
+          echo "diagnostic=$(cat missing_optypes.txt)" >> $GITHUB_OUTPUT
 
-  notify-slack:
-    uses: CQCL/hugrverse-actions/.github/workflows/slack-notifier.yml@main
+  create-issue:
+    uses: CQCL/hugrverse-actions/.github/workflows/create-issue.yml@main
     needs: missing-optypes
     if: ${{ needs.missing-optypes.outputs.should_notify == 'true' && ( github.event_name == 'schedule' || github.event_name == 'push' ) }}
     with:
-      channel-id: 'C040CRWH9FF'
-      slack-message: |
-        ⚠️ `tket-json-rs` is missing OpType definitions.
-        See <https://github.com/CQCL/tket-json-rs/actions/runs/${{ github.run_id }}|the failing check> for more info.
-      # Rate-limit the message to once per week
-      timeout-minutes: 10080
-      # A repository variable used to store the last message timestamp.
-      timeout-variable: "MISSING_OPS_MSG_SENT"
+        title: "`tket-json-rs` is missing OpType definitions."
```

Comment:
It would be nice (though not essential) to include the names of the missing optypes in the issue title.

Reply:
That requires a bit of piping and filtering the output of the test executable.
I'll make an issue so we can change it later.